### ISSUE TITLE
Hyperphantasia

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#11445]
**Preferred term label:**
Hyperphantasia

**Definition (free text, please give PubMed ID)**
Exceptionally vivid voluntary visual mental imagery, in which internally generated images are experienced as vivid as real seeing, despite intact reality testing and the absence of sensory hallucinations. (PMID:32446532; 34296179)

**Parent term (use hpo.jax.org/app)**
Abnormality of mental imagery
(a new term has been requested [here](https://github.com/obophenotype/human-phenotype-ontology/issues/11443))

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun added as HP:0020346, Thanks!
